### ISSUE TITLE
Invalid SVG-Export when using Layout.Frameless()

### ISSUE THREAD

**Issue by wnuessel** [ScottPlot/ScottPlot#4167]
Hello ScottPlot-Team,

thank you for providing ScottPlot-Library and for all of your work!
Now I have a problem when creating .svg-Images and I think it is a bug.

**Issue:**
When creating a .svg-File, the SVG-XML is invalid. There is missing a closing-Tag `</g>` at the end. It only happens, when setting Layout.Frameless()

As a result, Image-Viewer-Tools can not visualise the SVGs. I have tested with the MS-Edge-Browser.

**ScottPlot Version:**
V. 5.0.34: it worked fine
V. 5.0.35: first Release with that error
V. 5.0.37: error still exists

**Code Sample:**

			Plot myPlot = new Plot();

			double[] xs = { 1, 2, 3, 4, 5 };
			double[] ys = { 1, 4, 9, 16, 25 };

			var sl = myPlot.Add.ScatterLine(xs, ys, Colors.Black);
			sl.LineWidth = 2;
			sl.FillY = true;
			sl.FillYColor = Colors.Orange;

			// Frameless() leads to an invalid .svg-File
			myPlot.Layout.Frameless();

			myPlot.HideGrid();

			myPlot.SaveSvg("quickstart.svg", 400, 300);

**Here are the results of .svg-Files:**

V. 5.0.34: it worked fine:

```
<?xml version="1.0" encoding="utf-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400" height="300">
	<rect width="400" height="300"/>
	<rect fill="white" width="400" height="300"/>
	<clipPath id="cl_11">
		<rect y="19.565216" width="400" height="260.86957"/>
	</clipPath>
	<g clip-path="url(#cl_11)">
		<path fill="#FFA500" d="M18.1818 270L109.091 238.696L200 186.522L290.909 113.478L381.818 19.5652L381.818 280.435L18.1818 280.435"/>
	</g>
	<clipPath id="cl_12">
		<rect y="270" width="400" height="10.434784"/>
	</clipPath>
	<g clip-path="url(#cl_12)">
		<path fill="#FFA500" d="M18.1818 270L109.091 238.696L200 186.522L290.909 113.478L381.818 19.5652L381.818 280.435L18.1818 280.435"/>
	</g>
	<path fill="none" stroke="black" stroke-width="2" stroke-miterlimit="4" d="M18.1818 270L109.091 238.696L200 186.522L290.909 113.478L381.818 19.5652"/>
</svg>
```

With V. 5.0.37 there is missing a closing-Tag which is opened in Line
`<g clip-path="url(#cl_d)">`

```
<?xml version="1.0" encoding="utf-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400" height="300">
	<rect fill="white" width="400" height="300"/>
	<clipPath id="cl_d">
		<rect width="400" height="300"/>
	</clipPath>
	<g clip-path="url(#cl_d)">
		<clipPath id="cl_e">
			<rect y="19.565216" width="400" height="260.86957"/>
		</clipPath>
		<g clip-path="url(#cl_e)">
			<path fill="#FFA500" d="M18.1818 270L109.091 238.696L200 186.522L290.909 113.478L381.818 19.5652L381.818 280.435L18.1818 280.435"/>
		</g>
		<path fill="none" stroke="black" stroke-width="2" stroke-miterlimit="4" d="M18.1818 270L109.091 238.696L200 186.522L290.909 113.478L381.818 19.5652"/>
</svg>
```

If you need more information, let me know.

Regards,
Wolfgang

**Comment by aespitia** [comment]
this PR might fix it
https://github.com/ScottPlot/ScottPlot/pull/4121

**Comment by aespitia** [comment]
the update was merged in, but not yet released, i believe.

**Comment by wnuessel** [comment]
ok, I will test with next releases and close the issue, if fixed.

**Comment by wnuessel** [comment]
Bug is fixed with 5.0.38
Thank you!
